integration-test/src/main/java/com/sequenceiq/it/cloudbreak/dto/sdx/SdxInternalTestDto.java

```diff
@@ -346,6 +346,15 @@ public SdxInternalTestDto awaitForHealthyInstances() {
         return awaitForInstance(instanceStatusMap);
     }
 
+    public SdxInternalTestDto awaitForDatalakeServiceInitialization() {
+        try {
+            Thread.sleep(60000);
```

Comment:
It looks like a magic number. Can't we have a more precise wait logic here so that it won't be a flaky test in the future?

Reply:
I think this could be only a symptomatic treatment and could not solve the real issue. So maybe not the best way to choose. And later can cause also flakiness or constant failures, for example we got a related CM or API change and `60000` won't be enough...etc. 

Can we introduce some wait along with the API operation or flow? For example:
- all the necessary DB tables and operations have been done,
- or one or more CM service(s) is up and running (SOlr, Hbase, Atlas)
- introduce a wait operation to the API. So we can use this optionally like await for flow.